### PR TITLE
[WIP] Use factories in cucumber features

### PR DESCRIPTION
**Pull request by roschaefer**

### CODE REVIEW

**Comment by appinteractive** [comment]
@roschaefer I can`t run the tests locally anymore as it states that there is an authentication error with Neo4j. Any idea?

**Comment by roschaefer** [comment]
@appinteractive what exactly raises the authentication error? The factories or the UI? This PR is not yet finished, I forgot to add `[WIP]`.

File: cypress/integration/03.TagsAndCategories.feature [hunk 1]
```diff
@@ -14,28 +14,27 @@ Feature: Tags and Categories
...
-      | # | Name        | Nutzer | Beiträge |
-      | 1 | Naturschutz | 2      | 2        |
-      | 2 | Freiheit    | 2      | 2        |
-      | 3 | Umwelt      | 1      | 1        |
-      | 4 | Demokratie  | 1      | 1        |
+    Then I can see a list of tags ordered by user count:
+      | # | Name      | Users  | Posts    |
+      | 1 | Democracy | 2      | 3        |
+      | 2 | Ecology   | 1      | 1        |
+      | 3 | Nature    | 1      | 2        |
```
Comment (roschaefer) [inline cypress/integration/03.TagsAndCategories.feature]: @appinteractive cucumber features are now based on factories. That sometimes exposes strange behaviour: Why is this table not ordered by user **and** post count?

Comment (appinteractive) [reply cypress/integration/03.TagsAndCategories.feature]: Good question, I think that was not possible at that time to sort by more then one param.

**Pull request merged**